#! /bin/sh /usr/share/dpatch/dpatch-run
## 20_fix_manpage_name.dpatch by Sandro Tosi <matrixhasu@gmail.com>
##
## DP: Fix the upstream manpage for a valid first line

@DPATCH@
diff -urNad wmclock~/wmclock.man.in wmclock/wmclock.man.in
--- wmclock~/wmclock.man.in	2007-12-27 19:27:36.512023741 +0100
+++ wmclock/wmclock.man.in	2007-12-27 19:27:49.856784215 +0100
@@ -25,9 +25,7 @@
 .TH wmclock 1x "@DATE@" "Version @VERSION@"
 .\" ----------------------------------------------------------------
 .SH NAME
-.PP
-.BR wmclock :
-a dockable clock for the Window Maker window manager
+wmclock \- A dockable clock for the Window Maker window manager
 .\" ----------------------------------------------------------------
 .SH SYNOPSIS
 .HP
